doc/ref_model/chapters/appendix-a.md

```diff
@@ -70,6 +71,22 @@ Without arguing for or against SR-IOV, CNTT provides the following anecdotes whi
 End of policies.
 
 <a name="A.6"></a>
+## A.6 VNF Design and Implementation Requirements
+This section specifies a set of VNF Design and Implementation Requirements. 
+
+| Ref | Requirmenet | Unit | Definition
+|----|-----|-----|----|
+| vnf.des.001 | The VNF Developer **must** ensure that their software and the resultant VNF image does not contain malicious code (e.g., malware, logic bombs, etc.)  |  | |
+| vnf.des.002 | The VNF Developer **must** ensure that their software and the resultant VNF image does not contain code such as daemons that exposes them to risk |  | |
```

Comment:
```suggestion
| vnf.des.002 | The VNF Developer **should** ensure that their software and the resultant VNF image does not contain code such as daemons that exposes them to risk |  | |
```

Reply:
@kedmison Why should this not be required? Operators could ask for Vendors to certify to this.

Reply:
Indeed operators can do this.  But if it is specified here as a must, then it has to be flowed through to RI/RC as a requirement to test.

Perhaps this is best left as a 'should', and operators can make some of these mandatory during their procurement cycles if so desired.